File: programmers/src/stage1/java/나누어떨어지는숫자배열.java
package stage1;

import java.util.*;

public class 나누어떨어지는숫자배열 {
	public int[] solution(int[] arr, int divisor) {
		int[] answer = Arrays.stream(arr).filter(x -> x % divisor == 0).toArray();
		if (answer.length == 0) {
			answer = new int[1];
			answer[0] = -1;
		}
		Arrays.sort(answer);
		return answer;
	}
}

File: programmers/src/stage1/python/나누어떨어지는숫자배열.py
def solution(arr, divisor):
    answer = []
    for i in arr:
        # 나누어 떨어질 때 요소 추가
        if i % divisor == 0:
            answer.append(i)
    # 정렬 or [-1]
    answer.sort() if len(answer) != 0 else answer.append(-1)
    return answer
